plasma_cash/client/history.py

```diff
@@ -0,0 +1,34 @@
+import rlp
+from rlp.sedes import big_endian_int, binary, CountableList
+
+from .exceptions import TxHistoryNotCompleteException, TxHistoryNotFoundException
+
+
+class History(rlp.Serializable):
+
+    fields = [
+        ('deposit_tx_blk_num', big_endian_int),
+        ('latest_tx_blk_num', big_endian_int),
+        ('tx_history', CountableList(binary)),
+        ('proves', CountableList(binary))
+    ]
+
+    def __init__(self, deposit_tx_blk_num, tx, proof):
+        self.deposit_tx_blk_num = deposit_tx_blk_num
+        self.latest_tx_blk_num = deposit_tx_blk_num
+        self.tx_history = [tx]
+        self.proves = [proof]
+
+    def update_tx_histories(self, latest_tx_blk_num, txs, proves):
+        data_len = latest_tx_blk_num - self.latest_tx_blk_num
```

Comment:
I think the logic here might be wrong.
`latest_tx_blk_num` is used to trace the latest block with "inclusive" proof. However, "exclusive" proofs should be included to the proofs as well. When "exclusive" proofs are included, the logic here looks not right to me.

Reply:
Ok, I got it.